rest-service/manager_rest/rest/rest_utils.py

```diff
@@ -527,11 +527,11 @@ def retrieve_dependent_deployments(self, target_id):
         return results
 
     def retrieve_and_display_dependencies(self, target_id,
-                                          excluded_component_creator_id=None):
+                                          excluded_component_creator_ids=[]):
```

Comment:
Mutable defaults are the source of all evil
https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil